content/intro-to-storybook/angular/en/data.md

```diff
@@ -5,23 +5,23 @@ description: 'Learn how to wire in data to your UI component'
 commit: '0bf4edf'
 ---
 
-So far we created isolated stateless components –great for Storybook, but ultimately not useful until we give them some data in our app.
+So far, we have created isolated stateless components –great for Storybook, but ultimately not helpful until we give them some data in our app.
```

Comment:
Not sure about house style for em-dashes, but this should either be:

- No space: "...components—great..."
- Space: "...components — great..."
- Thin space 😍: "...components — great..."

Reply:
This is worth a find-and-replace, as I see a few examples of inconsistent formatting for this.

Reply:
I'm partial to no space, and updated it